components/shell/bash/Makefile

```diff
@@ -37,7 +37,7 @@ COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_PROJECT_URL=	http://www.gnu.org/software/bash/
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH = \
-    sha256:b4a80f2ac66170b2913efbfb9f2594f1f76c7b1afd11f799e22035d63077fb4d
+	sha256:b4a80f2ac66170b2913efbfb9f2594f1f76c7b1afd11f799e22035d63077fb4d
```

Comment:
Have something gone wrong????

Reply:
It's really easy to miss out new patches. Thanks for pointing that out.

I did not bump versioning as it seems to be doing in the Makefile by counting `00X` patches.

Reply:
At least, bump COMPONENT_REVISION